_sips/sips/2017-11-20-byname-implicits.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: sip
-title: SIP-36 - Byname implicit arguments
+title: SIP-31 - Byname implicit arguments
```

Comment:
Wasn't there any 31 before? :thinking:

Reply:
I couldn’t find it.

`$ grep -r "SIP-31" _sips/sips/` returned no match.